brizo/routes.py

```diff
@@ -187,12 +187,7 @@ def consume():
 
 @services.route('/compute', methods=['POST'])
 def compute():
-    """Allows to execute an algorithm inside in a docker instance in the cloud aquarius.
-
-
-    If the publisher of the assets
-    provide this service in the Service agreement related with the asset_did requested.
-
+    """Allows to execute an algorithm inside a Docker instance in the cloud aquarius. If the publisher of the assets provides this service in the service agreement related with the asset_did requested.
```

Comment:
> a Docker instance in the cloud aquarius

Not entirely clear what that means. What's a "cloud aquarius" and why is a Docker instance running within it?

Reply:
I think this used to say "Allows to execute an algorithm inside a Docker instance in the cloud aquarius." and then someone did a global search-and-replace of `provider` --> `aquarius`

I think the word `aquarius` could just be removed from the end of that sentence, since one runs the Docker instance "in the cloud" (e.g. Azure or whatever).

Reply:
ok, changed to something how I understood the sentence in 
https://github.com/oceanprotocol/brizo/pull/27/commits/f9b9695ed64feda0508f919e0c8fe2092d62a9f5